### PR TITLE
feature/case-studies

### PR DESCRIPTION
**Pull request by EricWaight**
**PR Includes:**
- Completed case study components
- Completed case study content

**Things to note:**
- This includes several slight deviations from original mockups provided, essentially to make things look/work better
- Case study content needs reviewed for accuracy
- Additional tech list icons could be added at a later time ([Jira task](https://b2io.atlassian.net/browse/B2-184) created for this)
- Case studies are missing horizontal scroll carousel for screenshots ([Jira task](https://b2io.atlassian.net/browse/B2-27) created for this). the idea was to get the case studies pushed out and spend time on adding this later

### CODE REVIEW

**Review by mpcondo — COMMENTED** [review]
This is looking really good.  I've looked over the code, and spelling/grammar of all the copy.  The individual case studies will still need to be reviewed for accuracy before it can be approved and merged.

File: src/components/BatteryXTPage/index.tsx [hunk 1]
```diff
@@ -1,12 +1,98 @@
...
+  TechListGrid,
+} from '~/components';
+import { spacing } from '~/theme';
 
 export const BatteryXTPage: FC = () => {
-  // TODO: Add description for BatteryXTPage.
+  const CaseStudyResultsResultsProps = {
+    children: [
+      {
+        heading: 'Generate inbound leads',
```
Comment (tborres) [inline src/components/BatteryXTPage/index.tsx:20]: All of the other leading verbs end with an `s`
```suggestion
        heading: 'Generates inbound leads',
```

File: src/components/CaseStudy/CaseStudyHeaderPane.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+type CaseStudyProps = NextImageProps & {
+  name: string;
+  src: Exclude<NextImageProps['src'], string | StaticImageData>;
+  alt: string;
+};
+
+export const CaseStudyHeaderPane: FC<CaseStudyProps> = ({ alt, name, src }) => {
+  return (
+    <section>
+      <Heading
```
Comment (tborres) [inline src/components/CaseStudy/CaseStudyHeaderPane.tsx:19]: 🎨 I think that we need to bump this h1 down to overlap the image (like it does in the comps) or move it up. The way it lands now looks odd to me.

**Current**
image

**Comp**
image

**Review by tborres — COMMENTED** [review]
I pushed a handful of changes, but I haven't yet finished reviewing everything. I ran out of time for the day.

**Comment by chriswhawkins** [comment]
Capturing everything I catch in a visual review of work and case-studies pages, specifically BatterXT
<hr />
- [ ] content clipped on work page
image
image
<hr />
- [ ] add D3js to technologies (https://github.com/d3/d3-logo)
image 
<hr />
- [ ] some weird user-select going on in this screenshot 
image
<hr />
- [ ] tough contrast ratio on the heading/image 
image
<hr />
- [ ] Long names can be clipped 
image
<hr />
- [ ] text gets clipped
image
<hr />
- [ ] wrong screenshot shown
image
<hr />

**Comment by k8hansen** [comment]
Since we do not have an immidate solution for the latin text on the Colorsnap page, I suggest we replace that section with some images. I'm not sure if we have an existing component that we can plug into that page to just handle images? If not, it might be worth making one for case studies that we do not have metrics for, and we can just substitute some css masonry or good looking grid images. 

I've taken some screenshots and put them [here](https://drive.google.com/file/d/18yQ3pnQ6jJnUHUs-zpR9bP-FO3ecUm6_/view?usp=sharing). I can help with this component if this is the route we want to take.

The only other issue that I see is that this subheader needs some more left margin for tablet and laptop screen sizes:

Screen Shot 2022-08-18 at 9 25 33 AM

**Review by k8hansen — APPROVED** [review]

**Pull request merged**